CMR/python/graphql/graphqlPaging.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import requests
+# Iterate over pages of graphql retrieving the search after cursor to do so
+token = os.getenv('TOKEN')
+# graphql-address e.g. 'https://graphql.sit.earthdata.nasa.gov/api'
+endpoint = os.getenv('GRAPHQL_URL')
+# Bearer may be needed before the token if you are using a personal token
+headers = {"Authorization": f"{token}"}
+cursor = ""
+page_num = 0
+query = """query ($params: CollectionsInput) {
+  collections(params: $params) {
+    cursor
+    items {
+      conceptId
+    }
+  }
+}"""
+
+variables = """{
+  "params":{
+    "limit":1,
+    "cursor":"%s"
+  }
+}"""% (cursor)
```

Comment:
does pylint complain about this older syntax? You use the modern "f" syntax on line 9 but not here. if pylint says nothing then your fine.